config/routes.rb

```diff
@@ -2,58 +2,58 @@
 require 'sidekiq-status/web'
 
 Rails.application.routes.draw do
-  scope '/:resource_name/:resource_id', constraints: { resource_name: /(questions|answers|comments)/ } do
-    post '/upvote', to: 'votes#upvote'
-    post '/downvote', to: 'votes#downvote'
-    resources :comments, except: [:new, :edit], resource_name: /(?!comments).*/
-  end
-
-  post '/validate_token', to: 'tokens#validate'
-
-  resources :questions, except: [:new, :edit] do
-    collection do
-      get :search
-    end
-    member do
-      get "recent_answers"
-      get "popular_answers"
+    scope '/:resource_name/:resource_id', constraints: { resource_name: /(questions|answers|comments)/ } do
+      post '/upvote', to: 'votes#upvote'
+      post '/downvote', to: 'votes#downvote'
+      resources :comments, except: [:new, :edit], resource_name: /(?!comments).*/
     end
 
-    resources :answers, except: [:new, :edit] do
+    post '/validate_token', to: 'tokens#validate'
+
+    resources :questions, except: [:new, :edit] do
+      collection do
+        get :search
+      end
       member do
-        post "accept"
+        get "recent_answers"
+        get "popular_answers"
+      end
+
+      resources :answers, except: [:new, :edit] do
+        member do
+          post "accept"
+        end
       end
     end
-  end
 
-  get "top_questions" => "questions#top_questions"
+    get "top_questions" => "questions#top_questions"
 
-  resources :users, only: [:show, :index] do
-    member do
-      get :questions
-      get :tags
-    end
+    resources :users, only: [:show, :index, :update] do
```

Comment:
Yea but this; we were talking of removing the `update action` since it doesn't have any practical public use.